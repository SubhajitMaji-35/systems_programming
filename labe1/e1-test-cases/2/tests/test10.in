100 -100 50 -50
25 -25 75 -75
10 -10 0 20
-1 1 -2 2
