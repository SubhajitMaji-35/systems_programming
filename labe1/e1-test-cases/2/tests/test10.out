100 25 10 -1
-100 -25 -10 1
50 75 0 -2
-50 -75 20 2
